defaults/main.yml

```diff
@@ -14,27 +14,31 @@ node_exporter_user: prometheus
 node_exporter_group: prometheus
 
 # manager service
-node_exporter_manage_service: True
+node_exporter_manage_service: true
 # start on boot
-node_exporter_service_enabled: True
+node_exporter_service_enabled: true
 # current state: started, stopped
 node_exporter_service_state: started
 
 # Files & Paths
+node_exporter_environment_path: /etc/default
 node_exporter_log_path: /var/log/node_exporter
 node_exporter_log_file: "node_exporter.log"
 node_exporter_root_path: /opt/node_exporter
 node_exporter_bin_path: "{{ node_exporter_root_path }}/bin"
-node_exporter_textfile_collector: "{{ node_exporter_root_path }}/textfile_collector"
+
+# Log
+node_exporter_log_level: warn  # [debug, info, warn, error, fatal]
 
 # Port & path
+node_exporter_host_name: "{{ ansible_nodename }}"
 node_exporter_port: 9100
-node_exporter_path: "metrics"
-
-node_exporter_options:
```

Comment:
The next release should be a new major version

Reply:
yep